packages/gasket-plugin-manifest/lib/utils.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
+const baseConfig = require('./base-config');
+
+/**
+ * Merges manifest defaults with gasket.config manifest and passes to
+ * every defined `manifest` hook for further manipulation.
+ *
+ * @param  {Gasket} gasket The gasket API
+ * @param {Object} context TODO
+ * @returns {Object} manifest.json
+ */
+async function gatherManifestData(gasket, context) {
+  const { logger, execWaterfall, config } = gasket;
+  const source = (context.req && context.req.originalUrl) || 'static manifest';
+
+  logger.debug(`Gathering manifest for ${source}`);
+
+  const fromConfig = config.manifest;
+  const manifest = { ...baseConfig, ...fromConfig };
```

Comment:
Make sure that the `staticOutput` property does not blend into the final `manifest.json` content. Add a test for this.